ksql-parser/src/main/java/io/confluent/ksql/parser/AstBuilder.java

```diff
@@ -1129,11 +1129,7 @@ public Node visitSubqueryExpression(SqlBaseParser.SubqueryExpressionContext cont
   @Override
   public Node visitDereference(SqlBaseParser.DereferenceContext context) {
```

Comment:
To clarify my understanding here:

If I have a statement like:
```
SELECT a.b.c from d
```
Then I should get something like:

DereferenceExpression(c)->DereferenceExpression(b)->DereferenceExpression(a)->QualifiedNameReference(d)

If DerefenceExpression(f)->e means a dereference of field f in expression e